packages/auth/src/auth-store.ts

```diff
@@ -115,7 +115,6 @@ export class AuthStore implements Signer {
     lifetime = MONTH_IN_SEC,
   ): Promise<ucan.Ucan> {
     const proofs: CapWithProof[] = []
-    const encodedTokens = new Set()
```

Comment:
was this just bad logic on my part 🤔 

the point of the logic here was to find the vaguest proof for each cape but to only encode each proof once so `encodedTokens` was meant to prevent duplication. but it looks like i either didn't finish it or it got merge-conflicted at some point.

the current code will work, but may result in some unnecessarily large UCANs. probably worth sticking a TODO in

Reply:
added in TODO myself because this will be an easy thing to forget about